receiver/splunkhecreceiver/config.go

```diff
@@ -35,10 +36,31 @@ type Config struct {
 	// Path we will listen on, defaults to `*` (anything matches)
 	Path     string `mapstructure:"path"`
 	pathGlob glob.Glob
+	// SourceKey informs the receiver to map the source field to a specific unified model attribute.
+	SourceKey string `mapstructure:"source_key"`
+	// SourceTypeKey informs the receiver to map the sourcetype field to a specific unified model attribute.
+	SourceTypeKey string `mapstructure:"sourcetype_key"`
+	// IndexKey informs the receiver to map the index field to a specific unified model attribute.
+	IndexKey string `mapstructure:"index_key"`
+	// HostKey informs the receiver to map the host field to a specific unified model attribute.
+	HostKey string `mapstructure:"host_key"`
 }
 
 // initialize and initialize the configuration
 func (c *Config) initialize() error {
+	if c.SourceKey == "" {
+		c.SourceKey = splunk.DefaultSourceLabel
+	}
+	if c.SourceTypeKey == "" {
+		c.SourceTypeKey = splunk.DefaultSourcetypeLabel
```

Comment:
nit: let's call it `DefaultSourceTypeLabel` not `DefaultSourcetypeLabel` for consistency with other similar names?

Reply:
Sure thing.